undertow/WFLY-9724_Allow_unescaped_characters_in_URL.adoc

```diff
@@ -0,0 +1,46 @@
+= WFLY-9724 Allow unescaped characters in URL
+:author:            Stuart Douglas
+:email:             sdouglas@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org
+
+== Overview
+
+This feature is to add an additional an addition option into the Undertow subsystem configuration to allow unescaped
+characters to be processed directly.
+
+== Issue Metadata
+
+=== Issue:
+
+* {issue-base-url}/WFLY-9724[WFLY-9724]
+
+=== Related Issues:
+
+* {issue-base-url}/EAP7-965[EAP7-965]
+
+=== Dev Contacts:
+
+* mailto:{email}[{author}]
+
+=== QE Contacts:
+
+=== Affected Projects or Components:
+
+* Wildfly
+
+== Requirements
+
+This will be implemented as an additional listener option under the Undertow subsystem called
+`allow-unescaped-characters-in-url`. This will be false by default for security reasons, however if it is set to true
+then this will let Undertow process unescaped non ASCII characters in the URL.
+
+=== Test Plan
+
+A new test will be created that adds a new listener with this option enabled. A request with UTF-8 in the URL
+will be sent to both the new listener and the current listener, it is expected that the new listener will reject it
+and the existing listener will process it.
```

Comment:
hm, I think the test behavior should be vice versa, that means: new listener (`allow-unescaped-characters-in-url` = true) will process that request whether existing listener (`allow-unescaped-characters-in-url` = false) will reject it.

Reply:
Fixed, this test is now part of the pull request